st/clitests/awss3api_spec.py

```diff
@@ -1841,6 +1853,335 @@ def get_md5_sum(fname):
+        .execute_test().command_is_successful()
+
+#  Step 1.2: Create object list file
+object_list_file = create_object_list_file("obj_list.json", obj_list, "true")
+
+# Step 2: Test plain request with no parameters
+# Command:= aws s3api list-object-versions --bucket <bucket>
+# Expected output:
+#   asdf, bo0, boo/0...boo/1, boo#, boo+, fo0, foo/0...foo/2, foo#123
+#   foo+123, qua0, quax/0...quax/3, quax#, quax+
+expected_versions_list = ['asdf', 'bo0', 'boo/0', 'boo/1', 'boo#', 'boo+',
+    'fo0', 'foo/0', 'foo/1', 'foo/2', 'foo#123', 'foo+123', 'qua0', 'quax/0',
+    'quax/1', 'quax/2', 'quax/3', 'quax#', 'quax+']
+list_versions_options = {}
+result = AwsTest('Aws list object versions with no parameters')\
+    .list_object_versions("versionlistbucket", **list_versions_options)\
+    .execute_test()\
+    .command_is_successful()
+
+# Process result set
+list_versions_response = get_aws_cli_object(result.status.stdout)
+if list_versions_response is None:
+    assert False, "Failed to list object versions"
+# Get versions from the response
+object_versions = list_versions_response["versions"]
+print(f"Response: {list_versions_response} Versions received: {object_versions}")
+check = all(item in object_versions for item in expected_versions_list)
+if check is False:
+    assert False, "Failed to match expected versions in the list"
+
+# Step 3: Test page-size = 2 (max-keys)
+# Command:= aws s3api list-object-versions --bucket <bucket> --page-size 2
+# Expected output:
+#   asdf, bo0, boo/0...boo/1, boo#, boo+, fo0, foo/0...foo/2, foo#123
+#   foo+123, qua0, quax/0...quax/3, quax#, quax+
+expected_versions_list = ['asdf', 'bo0', 'boo/0', 'boo/1', 'boo#', 'boo+',
+    'fo0', 'foo/0', 'foo/1', 'foo/2', 'foo#123', 'foo+123', 'qua0', 'quax/0',
+    'quax/1', 'quax/2', 'quax/3', 'quax#', 'quax+']
+list_versions_options = {"page-size":2}
+result = AwsTest('Aws list object versions with no parameters')\
+    .list_object_versions("versionlistbucket", **list_versions_options)\
+    .execute_test()\
+    .command_is_successful()
+
+# Process result set
+list_versions_response = get_aws_cli_object(result.status.stdout)
+if list_versions_response is None:
+    assert False, "Failed to list object versions"
+# Get versions from the response
+object_versions = list_versions_response["versions"]
+print(f"Response: {list_versions_response} Versions received: {object_versions}")
+check = all(item in object_versions for item in expected_versions_list)
+if check is False:
+    assert False, "Failed to match expected versions in the list"
+
+# Step 4: Test prefix which is not present
+# Command:= aws s3api list-object-versions --bucket <bucket> --prefix "abcd"
+# Expected output: No versions should be returned
+prefix = "abcd"
+list_versions_options = {"prefix":prefix}
+result = AwsTest(('Aws list object versions using prefix %s') % (prefix))\
+    .list_object_versions("versionlistbucket", **list_versions_options)\
```

Comment:
```suggestion
    .list_object_versions("versionlistbucket", prefix=prefix)\
```

Reply:
Changed the kwargs_options parameter to dictionary